.pipelines/vhd-builder-windows.yaml

```diff
@@ -1,12 +1,32 @@
+# Required pipeline variables:

```

Comment:
should we be running the pipeline on PRs?

Reply:
It looks like both the this and the Linux VHD pipeline builds are getting triggered for all PRs.
I was attempting to restrict PR triggered builds to only PRs that changes these two folders.

Would you prefer these only get triggered manually? If so I can update this to be pr:none.

Reply:
you can also just turn off triggers completely with `trigger: none`. What value is there with running this build for every pr?

Reply:
I already have trigger:none in this build definition and I don't think that trigger: none prevents the pipeline from running for new PRs
If you look at the linux VHD pipeline the yaml also has trigger: none and I see a bunch of builds trigger by PR 
https://msazure.visualstudio.com/One/_build?definitionId=35580&_a=summary many show the trigger as 'PR automated' and only a few show 'Manually triggered' as the cause.

I took a look at https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml and my understanding is that trigger: is used for pushing tags or commits to branches and pr: is used for whenever new pull requests created against branches.

My intention here was to reduce the amount of builds triggered by PRs from all PRs to just those changing files in specific directories.

If we want to turn off builds triggered by all PRs (which I'm ok with) i'll I can update to so both trigger:none and pr:none are set.

Reply:
It looks like the default behavior if pr: is not specified is to trigger builds for all PRs but @CecileRobertMichon showed me that the PR behavior for the Linux builds is overridden in the Azure DevOps build settings.
I'll update the .yaml file to add pr:none so by default no builds get triggered and then set finer grained controls in DevOps